.github/workflows/push.yml

```diff
@@ -571,7 +571,6 @@ jobs:
       id-token: write
     needs:
       - set-version
-      - release
```

Comment:
Nemělo by se to pustit až po buildu, kde jsou testy?

Reply:
Moze, ... myslel som, ze to mozeme doriesit aj neskor, testy na API este nemame ... checme to rozdelit do viacerych workflow. Opravene: https://github.com/keboola/keboola-as-code/pull/591/commits/9dc227e9dd949b7e81ba42711177df8a5ee6f1b2